decidim-admin/config/locales/en.yml

```
@@ -436,6 +436,12 @@ en:
         form:
           save: Save
         success: Help sections updated successfully
+      hidden_resource_mailer:
+        notify_mail:
+          body_1: An administrator removed your proposal because it has been reported as %{reason}
```

```suggestion
          body_1: An administrator removed your proposal because it has been reported
          body_2: 'Reason: %{reason}'
```

The reason might be a little bit impredicteable so maybe we should add a second line specifically for it